frontend/public/reducers/ui.ts

```diff
@@ -120,7 +120,7 @@ export default (state: UIState, action: UIAction): UIState => {
       }),
       queryBrowser: ImmutableMap({
         metrics: [],
-        pollInterval: null,
+        pollInterval: undefined,
```

Comment:
I try to avoid code that differentiates between `undefined` and `null`, which isn't very obvious and can be a source of bugs. I think this at least deserves a comment explaining.

Reply:
Thanks, the `null` / `undefined` distinction was not a great solution 😬

I'm now thinking that a better approach is to force the polling interval (`pollInterval={Math.round(timespan / 120)}`) instead of using `QueryBrowser`'s default polling interval logic. This removes the need for the `null` / `undefined` change.